data/xml/schema.rnc

```diff
@@ -12,8 +12,11 @@ MarkupText = (text | b | i | url | fixed-case | tex-math )+
 first = element first { text }
 last = element last { text }
 affiliation = element affiliation { text }
+orcid = element orcid { text }
+google-scholar = element google-scholar{ text }
+semantic-scholar = element semantic-scholar { text }
```

Comment:
What's the intended content of these fields? URLs (e.g. https://www.semanticscholar.org/author/Marcel-Bollmann/34887843) or IDs (34887843)?

If URLs, type should be `xsd:anyURI`, maybe even restricted by a pattern (`xsd:anyURI {pattern="https://www\.semanticscholar\.org/author/.+"}` (untested))

Reply:
Yeah, I'm going to wait on these. I agree they should be attached to authors, and then used for disambiguation.